### PR TITLE
added recipe for george

### PR DESCRIPTION
**Pull request by hellovenkat**
Recipe for george

### CODE REVIEW

File: recipes/george/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  summary: 'George is a fast and flexible Python library for Gaussian Process Regression.'
+  description: |
+    George is a fast and flexible Python library for Gaussian Process Regression.
+  doc_url: http://dan.iel.fm/george/current
+  dev_url: https://github.com/dfm/george
+
+extra:
+  recipe-maintainers:
+    - hellovenkat
+    - scopatz
```
Comment (synapticarbors) [inline recipes/george/meta.yaml]: @scopatz are you ok being listed as a co-maintainer of this recipe?

File: recipes/george/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  url: https://github.com/dfm/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/george/meta.yaml]: I think you need cython as a build depedency.

File: recipes/george/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/dfm/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
```
Comment (synapticarbors) [inline recipes/george/meta.yaml]: Why skip windows? It looks like `eigen` and the other dependencies are all available for that platform. I don't see anything in the upstream source repo that would indicate that it should be excluded.

File: recipes/george/meta.yaml (hunk 1 above)
Comment (scopatz) [inline recipes/george/meta.yaml]: Yes

File: recipes/george/meta.yaml (hunk 3 above)
Comment (scopatz) [inline recipes/george/meta.yaml]: Windows isn't a primary target for us. Folks are welcome to add support for it in the feedstock in the future, if they need it.

**Comment by scopatz** [comment]
Thanks @synapticarbors!

**Pull request merged**